src/ripple/rpc/handlers/Catalogue.cpp

```diff
@@ -1129,7 +1152,8 @@ doCatalogueLoad(RPC::JsonContext& context)
     jvResult[jss::ledger_count] =
         static_cast<Json::UInt>(header.max_ledger - header.min_ledger + 1);
     jvResult[jss::ledgers_loaded] = static_cast<Json::UInt>(ledgersLoaded);
-    jvResult[jss::file_size] = Json::UInt(file_size);
+    jvResult[jss::file_size_human] = formatFileSize(file_size);
+    jvResult[jss::file_size] = std::to_string(file_size);
```

Comment:
since we're here maybe we should add a projected filesize? this would be the average size of a delta multiplied by the number of deltas plus the size of the first ledger